index.html

```diff
@@ -210,6 +419,12 @@ <h2>
         "!ECMASCRIPT#sec-native-error-types-used-in-this-standard-typeerror">TypeError</dfn></code>
         is defined by [[ECMASCRIPT]].
       </p>
+      <p>
```

Comment:
Can delete the following. xref will take care of them.

Reply:
Indeed!